spec/triggers/gcp-pub-sub.md

```diff
@@ -0,0 +1,18 @@
+#  GCP PubSub Trigger
+
+This specification describes the `gcp-pub-sub` trigger.
+
+```yaml
+triggers:
+- type: gcp-pubsub
+  metadata:
+    subscriptionSize: "5"
+    subscriptionName: "mysubscription" # Required 
+    credentials: GOOGLE_APPLICATION_CREDENTIALS_JSON # Required
+```
+
+The GCP PubSub trigger allows you to scale based on the number of messages in your GCP PubSub subscription. The **credentials** property maps to the name of environment variable in the scale target (deployment) which contains the service account credentials (JSON) that would allow KEDA to connect to GCP and collect the required stack driver metrics to read the number of messages in the PubSub subscription. The **subscriptionSize** will be set as the target average value for the HPA and the **subscriptionName** is the subscription to be monitored.
```

Comment:
It's a bit of a big block to read, how about this?

```suggestion
The Google Cloud Platform‎ (GCP) Pub/Sub trigger allows you to scale based on the number of messages in your Google Cloud Platform‎ subscription.

The `credentials` property maps to the name of an environment variable in the scale target (`scaleTargetRef`) that contains the service account credentials (JSON). KEDA will use those to connect to Google Cloud Platform and collect the required stack driver metrics in order to read the number of messages in the Pub/Sub subscription.

`subscriptionName` defines the subscription that should be monitored. Based on the `subscriptionSize`, 
 the target average oe which the deployment will be scaled is defined.
```

Reply:
@tomkerkhove this should be fixed now